lessons/de/chapter_5.yaml

```
@@ -0,0 +1,224 @@
+- title: Mehrere Leben
+  content_markdown: >
+    Lebensdauerspezifizierer ermöglichen es uns, bestimmte Szenarien explizit zu definieren
+    Der Compiler kann sich nicht selbst auflösen
+
+    durch Unterscheidung aller Lebensdauern einer Funktionssignaturkomponente.
+  code: >-
+    https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&code=struct%20Foo%20%7B%0A%20%20%20%20x%3A%20i32%2C%0A%7D%0A%0A%2F%2F%20foo_b%20%C8%99i%20valoarea%20de%20retur%20au%20aceea%C8%99i%20durat%C4%83%20de%20via%C8%9B%C4%83%0A%2F%2F%20foo_a%20are%20o%20durat%C4%83%20de%20via%C8%9B%C4%83%20f%C4%83r%C4%83%20leg%C4%83tur%C4%83%20cu%20foo_b%0Afn%20do_something%3C'a%2C%20'b%3E(foo_a%3A%20%26'a%20Foo%2C%20foo_b%3A%20%26'b%20Foo)%20-%3E%20%26'b%20i32%20%7B%0A%20%20%20%20println!(%22%7B%7D%22%2C%20foo_a.x)%3B%0A%20%20%20%20println!(%22%7B%7D%22%2C%20foo_b.x)%3B%0A%20%20%20%20return%20%26foo_b.x%3B%0A%7D%0A%0Afn%20main()%20%7B%0A%20%20%20%20let%20foo_a%20%3D%20Foo%20%7B%20x%3A%2042%20%7D%3B%0A%20%20%20%20let%20foo_b%20%3D%20Foo%20%7B%20x%3A%2012%20%7D%3B%0A%20%20%20%20let%20x%20%3D%20do_something(%26foo_a%2C%20%26foo_b)%3B%0A%20%20%20%20%2F%2F%20se%20renun%C8%9B%C4%83%20la%20foo_a%20aici%20pentru%20c%C4%83%20numai%20durata%20de%20via%C8%9B%C4%83%0A%20%20%20%20%2F%2F%20a%20lui%20foo_b%20%C8%99i%20a%20valorii%20de%20retur%20continu%C4%83%20de%20aici%0A%20%20%20%20println!(%22%7B%7D%22%2C%20x)%3B%0A%20%20%20%20%2F%2F%20se%20renun%C8%9B%C4%83%20la%20x%20aici%0A%20%20%20%20%2F%2F%20se%20renun%C8%9B%C4%83%20la%20foo_b%20aici%0A%7D%0A
+- title: Statische Lebensdauern
+  content_markdown: >
+    Eine **statische** Variable ist eine Speicherressource, die zur Kompilierungszeit erstellt wird
+    existiert durch einen Programmstart
+
+    beenden. Ihre Typen müssen explizit angegeben werden.
+
+    
+    Eine **statische Lebensdauer** ist eine Speicherressource, die unbegrenzt vorhanden ist
+    Ende eines Programms. Beachten Sie, dass nach dieser Definition
+
+    Einige statische Lebenszeitressourcen können zur Laufzeit erstellt werden.
+
+    
+    Ressourcen mit statischer Lebensdauer haben einen speziellen Lebensdauerspezifizierer `'static`.
+
+
+    '`Statische' Ressourcen werden niemals **verloren**.
+
+
+    Wenn statische Lebenszeitressourcen Referenzen enthalten, müssen sie alle `'statisch` sein
+    (Alles andere würde nicht lange genug leben).
+
+
+    Speicherdetail:
+
+
+    * Das Ändern statischer Variablen ist von Natur aus gefährlich, weil sie es sind
+    weltweit zugänglich und für jedermann lesbar
+      Einführung der Möglichkeit eines Datenwettlaufs. Wir werden später über die Herausforderungen globaler Daten sprechen.
+    * Rust erlaubt die Verwendung von „unsafe { ... }“-Blöcken, um einige Operationen auszuführen
```

Use backticks for inline code:
```suggestion
    * Rust erlaubt die Verwendung von `unsafe { ... }`-Blöcken, um einige Operationen auszuführen
```